projects/plugins/jetpack/modules/subscriptions.php

```diff
@@ -973,6 +976,17 @@ public function register_post_meta() {
 	 * @return void
 	 */
 	public function add_subscribers_menu() {
+		/**
+		 * Enables the new in development subscribers in wp-admin dashboard.
+		 *
+		 * @since 9.5.0
```

Comment:
You can use `$$next-version$$` here ; it will be automatically replaced on release day.
https://github.com/Automattic/jetpack/blob/d3f58235992e84fcc8a4b1bc191c56ba6edf213a/projects/packages/README.md#L116-L123

```suggestion
		 * @since $$next-version$$
		 *
		 * @module subscriptions
```